README.md

```diff
@@ -35,7 +35,7 @@ Refers to files within the `_layouts` directory, that define the markup for your
 
   - `default.html` &mdash; The base layout that lays the foundation for subsequent layouts. The derived layouts inject their contents into this file at the line that says ` {{ content }} ` and are linked to this file via [FrontMatter](https://jekyllrb.com/docs/frontmatter/) declaration `layout: default`.
   - `home.html` &mdash; The layout for your landing-page / home-page / index-page.
-  - `page.html` &mdash; The layout for your documents that contain FrontMatter, but are not posts.
+  - `page.html` &mdash; The layout for your documents that contain FrontMatter, but are not posts. By default (unlike posts), page FrontMatter requires a `title` attribute, in order for a link to the page to be generated (in `header.html`).
```

Comment:
I see where you're coming from, but that doesn't address part of the underlying issue, that no title in Page FrontMatter, leads to... nothing, whereas with a Post, no title DWIM. Do you think your version is explicit enough that it's clear that *no* Page title => *no* output?